addons/mail/static/src/messaging/widget/discuss_invite_partner_dialog/discuss_invite_partner_dialog.js

```diff
@@ -26,12 +26,12 @@ const PartnerInviteDialog = Dialog.extend({
      * @param {Object} param1.messagingEnv.store
      */
     init(parent, { activeThreadLocalId, messagingEnv }) {
-        const store = messagingEnv.store;
-        const channelName = store.getters.threadName(activeThreadLocalId);
-        this.channelId = store.state.threads[activeThreadLocalId].id;
-        this.store = store;
+        const env = messagingEnv;
+        const channel = env.entities.Thread.get(activeThreadLocalId);
+        this.channelId = channel.id;
+        this.env = env;
```

Comment:
Same reasoning, if `channelId` can be retrieved from `env`, we probably should only store one of them.